orca-core/src/main/java/com/netflix/spinnaker/orca/exceptions/PipelineTemplateValidationException.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.exceptions;
+
+import com.netflix.spinnaker.kork.exceptions.HasAdditionalAttributes;
+import com.netflix.spinnaker.kork.exceptions.UserException;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+
+public class PipelineTemplateValidationException extends UserException
+    implements HasAdditionalAttributes {
+  private final Collection errors;
+
+  public PipelineTemplateValidationException(Collection errors) {
+    this.errors = errors;
+  }
+
+  public PipelineTemplateValidationException(String message, Collection errors) {
+    super(message);
+    this.errors = errors;
+  }
+
+  public PipelineTemplateValidationException(String message, Throwable cause, Collection errors) {
+    super(message, cause);
+    this.errors = errors;
+  }
+
+  public PipelineTemplateValidationException(Throwable cause, Collection errors) {
+    super(cause);
+    this.errors = errors;
+  }
+
+  public PipelineTemplateValidationException(
```

Comment:
Do we need all these constructors?

Reply:
silly copy-paste, thank you! removing a bunch of them